pcdsdevices/tests/test_lens_sets/test

```diff
@@ -1 +1 @@
-[2, 0.0002, 4, 0.0005]
```

Comment:
This is the file that will get repeatedly changed back and forth by pre-commit vs the test suite in an endless battle